scripts/release_scripts/update_configs.py

```diff
@@ -208,7 +208,7 @@ def verify_config_files(
     Args:
         release_feconf_path: str. The path to feconf file in release
             directory.
-        release_app_yaml_path: str. The path to app.yaml file in release
+        release_app_yaml_path: str. The path to app_dev.yaml file in release
```

Comment:
Can we also rename the variable?

```suggestion
        release_app_dev_yaml_path: str. The path to app_dev.yaml file in release
```

Reply:
Done.